mmv1/third_party/terraform/tests/resource_compute_service_attachment_test.go

```diff
@@ -25,7 +25,7 @@ func TestAccComputeServiceAttachment_serviceAttachmentBasicExampleUpdate(t *test
 				ResourceName:            "google_compute_service_attachment.psc_ilb_service_attachment",
 				ImportState:             true,
 				ImportStateVerify:       true,
-				ImportStateVerifyIgnore: []string{"target_service", "region"},
+				ImportStateVerifyIgnore: []string{"target_service", "region", "nat_subnets.0"},
```

Comment:
Why does this newly need to be set? Generally, having to set an ISVI value indicates that something's changed that will affect end users.

Reply:
Without this the test still fails with:
```
    provider_test.go:278: ImportStateVerify attributes not equivalent. Difference is shown below. Top is actual, bottom is expected.
        
        (map[string]string) (len=1) {
         (string) (len=13) "nat_subnets.0": (string) (len=121) "https://www.googleapis.com/compute/beta/projects/tjr-dm-test-1/regions/us-west2/subnetworks/tf-test-psc-ilb-natn86tr1bw0e"
        }
        
        
        (map[string]string) (len=1) {
         (string) (len=13) "nat_subnets.0": (string) (len=81) "projects/tjr-dm-test-1/regions/us-west2/subnetworks/tf-test-psc-ilb-natn86tr1bw0e"
        }
```
`nat_subnets` `Elem.DiffSuppressFunc` is `compareSelfLinkOrResourceName`, so I'm fairly confident this is a problem that only occurs in testing.